tests/script/test_ciphey.py

```diff
@@ -1,10 +1,22 @@
 # -*- coding: utf-8 -*-
 # This file is part of Quark-Engine - https://github.com/quark-engine/quark-engine
 # See the file 'LICENSE' for copying permission.
-
+from unittest import TestCase
+from unittest.mock import patch
 from quark.script.ciphey import checkClearText
 
 
+class TestWithoutCiphey(TestCase):
```

Comment:
To make the code look organized, could you add `testCheckClearTextWithClearText` and `testCheckClearTextWithCipherText` to the class?